### DISCUSSION TITLE
New Data Visualization feature

### DISCUSSION THREAD

**Discussion by CharlieSnowCode** [thunderclient/thunder-client-support#1454] [Q&A]
Has anyone used this successfully?  I have tried over a dozen different things but almost always get a blank output in the 'Response, Chart' area.  The one time I did get something it came through as literally `{{valueFromResponse}} `it did not translate the value.

**Comment by rangav** [comment]
See this example 
https://github.com/rangav/thunder-client-support/issues/511#issuecomment-1889916611
